src/main/java/ti4/model/TokenModel.java

```diff
@@ -81,7 +81,7 @@ public MessageEmbed getRepresentationEmbed() {
     @Override
     public boolean search(String searchString) {
         return getId().contains(searchString)
-            || getAliasList().toString().contains(searchString)
+            || (getAliasList() != null && getAliasList().toString().contains(searchString))
```

Comment:
Address errors in search attempts
- https://discord.com/channels/943410040369479690/1341438476528123904/1376380617411002483